tests/cases/conformance/externalModules/umd9.ts

```diff
@@ -0,0 +1,14 @@
+// @module: commonjs
+// @noImplicitReferences: true
+// @allowUmdGlobalAccess: true
+
+// @filename: foo.d.ts
+declare class Thing {
+  foo(): number;
+}
+export = Thing;
+export as namespace Foo;
+
+// @filename: a.ts
+/// <reference path="foo.d.ts" />

```

Comment:
Is this a requirement in **this file**?

Given the <a href="https://github.com/Microsoft/TypeScript/issues/10178">original issue <b>#1078</b></a> is sooo convoluted, it may be well worth a bit of write-up how this new feature works.

P.S. I have a feeling it may be extremely useful in some scenarios I am involved, but need a more defined behaviour/usage/spec to know for sure.

Reply:
You're right, the original issue is a lot to absorb, but this “feature” is actually quite simple. All it does is suppress a particular error message. This one:

![image](https://user-images.githubusercontent.com/3277153/56676223-b164f800-6672-11e9-9c9c-a1658c2e914f.png)

The triple slash reference isn't related to this feature at all; it's related to the `@noImplicitReferences` option which is just a configuration for our test harness.

Does that clear things up, or do you have any additional specific questions?

Reply:
Thanks, all clear.

It's awesome: from complex to trivial in a couple sentences.